.github/workflows/playwrightCI.yml

```diff
@@ -0,0 +1,44 @@
+name: playwright CI
```

Comment:
Could optionally combine jestCI and playwrightCI into testCI - would run a bit quicker but would need to dig more to see what was going wrong. I don't have a particular preference.